apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -239,6 +252,42 @@ export default connect(
           return style.debugOutputBackgroundWarning;
       }
     }
+    projection() {
+      return geoMercator()
+        .scale(60)
```

Comment:
Style: let's move these values to constants at the top to be able to find and change easily in the future: https://github.com/code-dot-org/code-dot-org/blob/46a5b30b9a2bb05c42addf8d93fa1236d18b84bb/apps/src/applab/PhoneFrame.jsx#L11

Reply:
Refactored!